content/articles/java-program-to-store-unicode-characters-using-character-literals/index.md

```diff
@@ -0,0 +1,129 @@
+    {       
+        char first1 = '\u0041';
+        System.out.println(first1);
+        char second2 = '\u03C0';
+        System.out.println(second2);
+        char third3 = '\u0031';
+        System.out.println(third3);
+        char fourth4 = '\u007E';
+        System.out.println(fourth4);
+        char fifth5 = '\u0024';
+        System.out.println(fifth5);
+        char sixth6 = '\u002F';
+        System.out.println(sixth6);
+        char seventh7 = '\u03C0';
+        System.out.println(seventh7);
+    }
+}
+```
+**OUTPUT**
+```
+/
+?
+1
+~
+$
+a
+A
+```
+3. Third method: Giving ASCII data types ASCII characters
+
+A single quote will save the ASCII value in a variable, which will then be used to print the value. The implementation is shown below.
+```Java
+import java.io.*;
+class Example {
+    public static void main(String[] args)
+    {
+        char first1 = 36;
+        System.out.println(first1);
+        char second2 = 47;
+        System.out.println(second2);
+        char third3 = 49;
+        System.out.println(third3);
+        char fourth4 = 126;
+        System.out.println(fourth4);
+        char fifth5 = 36;
+        System.out.println(fifth5);
+        char sixth6 = 97;
+        System.out.println(sixth6);
+        char seventh7 = 65;
+        System.out.println(seventh7);
+    }
+}
+```
+**OUTPUT**
+```
+/
+?
+1
+~
+$
+a
+A
+```
+In order to print Unicode characters, enter the escape sequence "u".
+
+Unicode sequences can be used everywhere in Java code. As long as it contains Unicode characters, it can be used as an identifier. Using Unicode, you may convey comments, IDs, and even the content of character and string literals, as well as other information. Use caution, as they are interpreted by the compiler early. Using the Unicode linefeed as part of a print statement would cause an error in your source code because your source code would treat this as an actual linefeed that appears before the closing single quote of a character literal, which would result in an error. 'n' and 'r' for line termination literals is the basis for the prior warning to always use these characters.
+
+### Advantages of unit code
+- Symbols for reading and writing character data are contained in a single code page when an application component employs Unicode. Applications may now be developed much more quickly thanks to this.
+- Each of the standard ASCII characters in UTF-8's first 127 places is assigned its ASCII value. For ASCII applications, this simplifies the conversion process.
+- It is possible for OpenEdge clients that utilize different code pages to read and write to a single UTF-8 database without any problems. With every transfer of data from client to database, OpenEdge transforms the code page to match.
+
+One disadvantage is that as a result of Unicode's substantially expanded alphabetic symbol set, it requires at least double the amount of memory to store a Roman alphabet character than ASCII does.
+### More on Literals 

```

Comment:
I suggest you also add an external link to another resource that might help the reader understand the topic deeper if you have any.

Reply:
Seems like you forgot to do this.